contracts/contracts/ERC677/mock/token677ReceiverMock.cairo

```diff
@@ -0,0 +1,85 @@
+%lang starknet
```

Comment:
What’s the purpose of this contract vs the mock/linkReceiver.cairo one? Do we really need both?

Reply:
I think this is because the link token repo has both types of mocks https://github.com/smartcontractkit/LinkToken/tree/f307ea6d4c02dd87ea4c1f1032e3e384ceac20c2/contracts/v0.6/mocks